src/Components/App/App.js

```diff
@@ -8,32 +8,35 @@ import './App.css';
 
 class App extends Component {
   constructor() {
-    super();
+    super()
     this.state = {
       movies: [],
       error: '',
       singleMovie: null,
       bannerImage: ''
-    };
+    }
   }
 
   componentDidMount = () => {
     fetchAllMoviesData()
```

Comment:
I haven't looked, and this might be the case already, but be sure to have error handling for bad responses from the server in your `apicalls` file. Good to see in here that you are putting the error in state and I'm guessing you're showing the error to your user.